src/services/itemMembership/index.ts

```diff
@@ -5,25 +5,18 @@ import { FastifyPluginAsync } from 'fastify';
 
 import { IdParam, PermissionLevel } from '@graasp/sdk';
 
-import { WEBSOCKETS_PLUGIN } from '../../utils/config';
 import { buildRepositories } from '../../utils/repositories';
-import { ItemMembership } from './entities/ItemMembership';
 import { PurgeBelowParam } from './interfaces/requests';
-import common, { create, createMany, deleteAll, deleteOne, getItems, updateOne } from './schemas';
+import common, { create, createMany, deleteOne, getItems, updateOne } from './schemas';
 import ItemMembershipService from './service';
-
-// import { registerItemMembershipWsHooks } from './ws/hooks';
+import { membershipWsHooks } from './ws/hooks';
 
 const ROUTES_PREFIX = '/item-memberships';
 
 const plugin: FastifyPluginAsync = async (fastify) => {
-  const {
-    db,
-    items: { service: itemService },
-    mailer,
-    hosts,
-  } = fastify;
-  const itemMembershipService = new ItemMembershipService(itemService, hosts, fastify.mailer);
+  const { db, items, mailer, hosts } = fastify;
+  const itemMembershipService = new ItemMembershipService(items.service, hosts, mailer);
+  items.membserships = { service: itemMembershipService };
```

Comment:
Considering the above, you would instead have:
```suggestion
  fastify.decorate('memberships', { service: itemMembershipService });
```
So it wouldn't depends on the item service being decorated. However I don't know if it breaks any scoping rules